src/inboxsdk-js/inboxsdk.ts

```diff
@@ -33,9 +34,9 @@ export function load(version: any, appId: string, opts: any) {
       VERSION: LOADER_VERSION,
       REQUESTED_API_VERSION: version,
     },
-  );
+  ) satisfies PiOpts;
```

Comment:
This should still be a `Partial<PiOpts>` because we still set some default properties further along at `makePlatformImplementation()`.

Oh, I see that `PlatformImplementationLoader.load()` is typed as taking in a `PiOpts`. We should make sure the type is set to `Partial<PiOpts>` up to it getting into `makePlatformImplementation()`.